web/src/components/app-survey/FeedbackCard.tsx

```diff
@@ -39,20 +51,26 @@ export default function FeedbackCard({
   postSurveyResponse,
   subtitle,
   primaryQuestion,
+  preceedingQuestion,
   secondaryQuestionHigh,
   secondaryQuestionLow,
   surveyReference,
 }: FeedbackCardProps) {
   const [isClosed, setIsClosed] = useState(false);
   const [feedbackState, setFeedbackState] = useState(FeedbackState.INITIAL);
   const setHasSeenSurveyCard = useSetAtom(hasSeenSurveyCardAtom);
+
+  const setHasSeenUsSurveyCard = useSetAtom(hasSeenUsSurveyCardAtom);
   const isMobile = useIsMobile();
 
   const handleClose = () => {
     setIsClosed(true);
     if (surveyReference === 'Map Survey') {
       setHasSeenSurveyCard(true);
     }
+    if (surveyReference === 'US Survey') {
+      setHasSeenUsSurveyCard(true);
+    }
```

Comment:
These are mutually exclusive right?

Should we use if else statements instead?

Reply:
Yeah they are mutually exclusive. I think it's a touch easier to read / extend without the else in there. In the case there is an overlap it could be nice that it closes them both